src/NavList/NavList.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import {ActionList} from '../ActionList'
+
+// ----------------------------------------------------------------------------
+// NavList
+
+export type NavListProps = {
+  children: React.ReactNode
+  'aria-label'?: string
+  'aria-labelledby'?: string
+  // sx
+}
+
+const Root = React.forwardRef<HTMLElement, NavListProps>(
+  ({'aria-label': ariaLabel, 'aria-labelledby': ariaLabelledBy, children}, ref) => {
+    return (
+      <nav ref={ref} aria-label={ariaLabel} aria-labelledby={ariaLabelledBy}>
+        <ActionList>{children}</ActionList>
+      </nav>
+    )
+  }
+)
+
+Root.displayName = 'NavList'
+
+// ----------------------------------------------------------------------------
+// NavList.Item
+
+export type NavListItemProps = {
+  children: React.ReactNode
+  href?: string
+  'aria-current'?: 'page' | 'step' | 'location' | 'date' | 'time' | 'true' | 'false' | boolean
+  // sx
+}
+
+const Item = React.forwardRef<HTMLAnchorElement, NavListItemProps>(
+  ({href, 'aria-current': ariaCurrent, children}, ref) => {
+    return (
+      <ActionList.LinkItem
+        ref={ref}
+        href={href}
+        aria-current={ariaCurrent}
+        sx={{
+          position: 'relative',
+          '&[aria-current]': {
```

Comment:
should this be in ActionList? other components like `AutoComplete` and `SelectPanel` would reuse the styles for active descendent

Reply:
Yeah, I think that makes sense. Gonna leave it here for now, then abstract it in a separate PR.